### ISSUE TITLE
Remove non-executive level domains from the Pulse Accessibility scan

### ISSUE THREAD

**Issue by smarina04** [18F/pulse#501]
Per OGP's decision as of 8/12/16, the accessibility scan should be limited to the executive level domains only.

**Comment by gbinal** [comment]
Roger.  This'll be included in #477.

**Comment by gbinal** [comment]
done

**Comment by konklone** [comment]
It's not done:

https://staging.pulse.cio.gov/accessibility/domains/#q=supremecourt

**Comment by gbinal** [comment]
Thank you, yes.  I'd intended to file this as a new issue since it looks like this is a data quality issue somewhere upstream.  

See - https://pulse.cio.gov/analytics/domains/#q=supreme and the [HTTPS CSV export](https://pulse.cio.gov/data/domains/https.csv).  It would seem that (some of?) the judicial agencies are being labeled as executive branch agencies upstream.

**Comment by gbinal** [comment]
this is addressed with an interim fix by https://github.com/18F/pulse/pull/550

**Comment by gbinal** [comment]
This is currently fixed in staging and should not recur.  However, I'd like to leave this issue open until we've gone through the process of running new scans and confirmed that it does not recur.

**Comment by gbinal** [comment]
Done
